packages/editor/src/components/collab-sidebar/style.scss

```diff
@@ -132,6 +123,16 @@
 	font-weight: 500;
 }
 
+.editor-collab-sidebar-panel__comment-form textarea {
+	@include input-control;
+	// Vertical padding is to match the standard 40px control height when rows=1,
+	// in conjunction with the 20px line-height.
+	// "Standard" metrics are 10px 12px, but subtracts 1px each to account for the border width.
+	padding: 9px 11px;
+	line-height: 20px !important;
```

Comment:
This code is taken from the TextAreaControl component: 

https://github.com/WordPress/gutenberg/blob/2de47fb98ba5b3293ca2cf6a15878b5738b2e30e/packages/components/src/textarea-control/styles/textarea-control-styles.ts#L39-L47

Reply:
Thanks for clarifying and thanks for adding the CSS comments, very useful for the context.